nodes/locales/en-US/clarify_api.html

```diff
@@ -1,12 +1,48 @@
 <script type="text/html" data-help-name="clarify_api">
   <p>Configuration node for Clarify API</p>
 
-  <h3>General Properties</h3>
-  <p>All properties are mandatory.</p>
+  Follow these steps to configure this node:
+  <ol>
+    <li>Go to www.clarifyapp.clarify.io and either log in or create a free account</li>
+    <li>Navigate to the admin panel by clicking your organization name in the upper right corner and click integrations</li>
+    <li>Choose an existing integration or create a new one</li>
+    <li>In the right side-panel click Access Control and + Create Client Credentials</li>
+    <li>Choose OAuth2 and give your credentials a name to remeber where your're using it, save the json file</li>
+    <li>Back in the node configuration (Node-Red) click Credentials: Choose File and upload the json file from Clarify </li>
+    <li>Click Add and deploy your flow. You can then return to the config and click Check Credentials to ensure that the token fetching is working (not necessarry)</li>
+  </ol>
+
+  <h4>You can find relevant guides in our developer documentation:</h4>
+  <a href="https://docs.clarify.io/developers/quickstart/create-integration">Creating an Integration</a><br>
+  <a href="https://docs.clarify.io/developers/tools/node-red/using-clarify-nodes">Using the Clarify Nodes</a>
+
+  <h3>Properties</h3>
+  <p>The only mandatory configuration is uploading the credentials file from Clarify</p>
+  <dl class="message-properties">
+    <dt>Name<span class="property-type">string</span></dt>
+    <dd>A name to distinguish the different configuration nodes</dd>
+
+    <dt>Credentials <span class="property-type">button</span></dt>
+    <dd>Button to upload your clarify-credentials.json file</dd>
+
+    <dt>Access token <span class="property-type">button</span></dt>
+    <dd>Check that the token fetching is working (flow deploy needed)</dd>
+
+    <dt>Clear cache<span class="property-type">button</span></dt>
+    <dd>Remove all signal cache from the local-database to enforce creation of signals and/or updating the signal meta-data</dd>
+  </dl>
+  <h3>Advanced Properties</h3>
+  <p>You should normally not need to change any of these settings. They are available for overriding when debugging</p>
   <dl class="message-properties">
-    <dt>Clarify API url <span class="property-type">string</span></dt>
+    <dt>API url <span class="property-type">string</span></dt>
     <dd>URL to the Clarify API Endpoint</dd>
 
+    <dt>Token url <span class="property-type">string</span></dt>
+    <dd>URL for OAuth2 token flow</dd>
+
+    <dt>Audience <span class="property-type">string</span></dt>
+    <dd>OAuth2 Audience</dd>
+
     <dt>Client ID <span class="property-type">string</span></dt>
     <dd>Client ID credential</dd>
```

Comment:
Do we still have this properties? Are these not read from the credentials file?

Reply:
I.e. you could debug by altering the credentials file if you really need to.

Reply:
They are available under the Advanced Properties tab:

![Screenshot 2022-02-16 at 13 45 06](https://user-images.githubusercontent.com/4091634/154267231-7c8cba5f-4ee0-45d7-85bf-932ff433757c.png)

You could edit the credentials file, but there is no way to override all settings from there. Not that this is needed anymore, so we can consider removing it later...

Reply:
Alright. Let's approve the updated docs, but I would be in favor of removing it all together.